model/flow/quorum_certificate.go

```diff
@@ -14,7 +14,7 @@ type QuorumCertificate struct {
 	// In order to distinguish the signature types, the SigData has to be deserialized. Specifically,
 	// the field `SigData.SigType` (bit vector) indicates for each signer which sig type they provided.
 	// For collection cluster, the SignerIDs includes all the staking sig signers.
-	SignerIDs []Identifier
+	SignerIndices []byte
```

Comment:
Q: why not combining `SignerIndices` and `SigData` since their type are both `[]byte`?

A: SigData contains signature, we don't want to validate signature before validating the signer IDs, because signature validation is expensive. 

Keeping them separated allows us to decode the signer indices alone, and check the total stakes represents the majority before decoding and validating SigData field.